201-winrm-windows-vm/azuredeploy-parameters.json

```diff
@@ -20,9 +23,6 @@
     "vaultName" : {
       "value" : ""
     },
-    "vaultResourceGroup" : {
```

Comment:
This is actually required when a vault is in a different resource group. Think about when you want to delete a resource group and it deletes all the resources within, you don't want to accidentally delete your vault and have to set it up again.